packages/roosterjs-editor-types/lib/event/SelectionChangeEvent.ts

```diff
@@ -0,0 +1,28 @@
+import BasePluginEvent from './BasePluginEvent';
+import { PluginEventType } from '../enum/PluginEventType';
+import { SelectionRangeEx } from '../interface/SelectionRangeEx';
+import type { CompatiblePluginEventType } from '../compatibleEnum/PluginEventType';
+
+/**
+ * Data of SelectionChangedEvent
+ */
+export interface SelectionChangedEventData {
+    /**
+     * Information of the selection
+     */
+    selectionRangeEx: SelectionRangeEx;
```

Comment:
How do you want to handle the case when unselect an image or a table?

In theory that will cause a normal selection to be created. But since that will be handled in separate PR, what will we pass for selectionRangeEx for now?